src/main/resources/swagger/api-docs.yaml

```diff
@@ -2037,7 +2177,7 @@ paths:
             {
               "type": "array",
               "items": {
-                "type": "string"
+                $ref: '#/definitions/BillingProjectMembership'
```

Comment:
Nitpick - there is no need for the json style formatting of shcema. Not Doug's code, but we should try to be consistent with the rest of the doc.